### ISSUE TITLE
Does it support bitbucket server?

### ISSUE THREAD

**Issue by Aouchinx** [git-touch/git-touch#240]
I have my own bitbucket server, and  i get the following error when i try to log in on the git-touch app.

![git-touch-issue](https://user-images.githubusercontent.com/71441807/120741283-5ecfb880-c527-11eb-81f0-e3723be3309f.png)

I have noticed the tip about App Password. I did not find this feature in Bitbucket Server. But I found a setting item for Personal access tokens. I created a token for git touch app, and entered the token, finally i got this format error.

![image](https://user-images.githubusercontent.com/71441807/120741569-d6054c80-c527-11eb-894b-84f0de0a6b11.png)

So, it doesn't support Bitbucket Server ? Or just a bug?

**Comment by shreyas1599** [comment]
@Aouchinx it's not supported yet. Bitbucket Cloud and Server API calls are different. The app only supports Cloud API calls.

**Comment by pd4d10** [comment]
Yeah, just as @shreyas1599 says, the recommended way of BitBucket API seems to be the [app passwords](https://support.atlassian.com/bitbucket-cloud/docs/app-passwords/). 

Looks like the case is a self-hosted BitBucket. Is there any documentation for it?

**Comment by shreyas1599** [comment]
@pd4d10 yeah there is. Its maintained separately and has a different version also. 
https://developer.atlassian.com/server/bitbucket/how-tos/command-line-rest/

[This](https://community.atlassian.com/t5/Bitbucket-questions/Does-the-rest-api-2-0-exist-for-server/qaq-p/881631) talks about how they're different

**Comment by pd4d10** [comment]
OK. I guess we can support it, but we may need a server for testing

**Comment by Aouchinx** [comment]
@pd4d10 @shreyas1599 
Okay, I already understand. Thank you both for letting me know the difference. 🤝
